Client/src/components/Shared/NotLoggedInView/NotLoggedInView.jsx

```
@@ -0,0 +1,81 @@
+        <h1 className="text-4xl sm:text-5xl md:text-6xl font-bold mb-4 leading-tight text-white">
+          Unlock the <span className="text-blue-400">emotional impact</span> of your video ads
+        </h1>
+        <h2 className="text-xl md:text-2xl mb-8 text-gray-300">Facial emotion detection for data-driven advertising</h2>
+        <p className="text-lg mb-10 text-gray-400">See how viewers really feel about your content. Optimize ad performance with EngageAnalytics.</p>
+        
+        <button 
+          onClick={handleGetStarted}
+          className="relative group px-8 py-4 bg-blue-600 text-white text-lg font-semibold rounded-full overflow-hidden shadow-lg"
+          type="button"
+        >
+          <span className="absolute inset-0 w-full h-full bg-gradient-to-r from-blue-500 to-blue-700 rounded-full shadow-md"></span>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)